Scripts/build.swift

```diff
@@ -213,33 +283,54 @@ enum Task: String, CustomStringConvertible {
 
 guard CommandLine.arguments.count > 2 else {
     print("Usage: build.swift platforms [spm|xcode]")
-    throw TaskError.code(1)
+    exit(0)
 }
 let rawPlatforms = CommandLine.arguments[1].components(separatedBy: ",")
 let rawTask = CommandLine.arguments[2]
 
 guard let task = Task(rawValue: rawTask) else {
     print("Received unknown task \(rawTask)")
-    throw TaskError.code(1)
+    exit(0)
+}
+
+let platforms = rawPlatforms.map { rawPlatform -> Platform in
+    guard let platform = Platform(rawValue: rawPlatform) else {
+        print("Received unknown platform type \(rawPlatform)")
+        exit(0)
+    }
+
+    return platform
 }
 
-if task.shouldGenerateXcodeProject {
+if task == .spm && platforms.map({ task.shouldGenerateXcodeProject(for: $0) }).contains(true) {
     try execute(commandPath: "/usr/bin/xcrun", arguments: ["/usr/bin/swift", "package", "generate-xcodeproj", "--output=generated/"])
 }
 
+for platform in platforms {
+    var deletedXcodeproj = false
+    var xcodeBuildArguments: [String] = []
+    if task == .spm && platform.requiresUpdatedSPMIntegration {
+        do {
+            print("Deleting Valet.xcodeproj, any uncommitted changes will be lost.")
```

Comment:
```suggestion
            // Swift will error if an xcodeproj and Package.swift file are both present in this directory. Delete the xcodeproj to prevent this. We'll restore it later.
            print("Deleting Valet.xcodeproj, any uncommitted changes will be lost.")
```

Reply:
I added some different comments which should resolve this.